### DISCUSSION TITLE
Basic drawing functions

### DISCUSSION THREAD

**Discussion by smorad** [deepmind/dm_pix#105] [Ideas]
Hello,

Thank you for the work on `pix`. I was wondering if there were any plans to implement basic primitives/drawing functions (circles, rectangles, polygons, etc), similar to the primitives from PIL.

**Comment by claudiofantacci** [comment]
hello there @smorad! Thanks a lot for the kind words and for your question.

We currently don't have plans for these primitives/drawings just yet I'm afraid.
If you have anything your end you're willing to contribute with, please feel free to submit a PR!
